### DISCUSSION TITLE
Run shell script from winebrowser.exe?

### DISCUSSION THREAD

**Discussion by emendelson** [The-Wineskin-Project/WineskinServer#367] [Q&A]
This question didn't get a usable answer at winehq, so I hope it's OK to ask it here.

My wrapped apps use winebrowser.exe to run AppleScript apps, and they work perfectly, with the syntax:

`winebrowser.exe file://path/to/myapp.app`

I can't find a way to run a shell script in the same way. I read at winehq that winebrowser.exe is registered in the wine registry to run `.app` applications, but I can't find that registry entry, and so can't find a way to register the. `.sh` scripts. Am I missing something obvious? I've spent a lot of time trying to solve this one, so I hope it's OK to ask for help.

**Comment by Gcenx** [comment]
I believe what your looking for is [How do I launch a native application from a windows application?](https://wiki.winehq.org/FAQ#How_do_I_launch_native_applications_from_a_Windows_application.3F)

**Reply by Gcenx** [reply]
Along with [How do I associate a native program with a file type in Wine?](https://wiki.winehq.org/FAQ#How_do_I_associate_a_native_program_with_a_file_type_in_Wine.3F)

**Reply by emendelson** [reply]
Thank you! That's pointing me in the right direction. I haven't got it working yet, but will get it eventually. Thank you again.
